client/services/ReinforcementGeneratorServiceSpec.js

```diff
@@ -0,0 +1,373 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests the ReinforcementGeneratorService
+ */
+
+describe('ReinforcementGeneratorService', function() {
+  // Task with 2 correctness tests
+  var testTask;
```

Comment:
It might be better to name these e.g. taskWithTwoTests or something more descriptive, so that the reader doesn't have to cross-reference so much to remember what '1', '2' and '3' map to.